javav2/example_code/rekognition/src/main/java/com/example/rekognition/VideoDetect.java

```diff
@@ -0,0 +1,286 @@
+
+            Video vidOb = Video.builder()
+                    .s3Object(s3Obj)
+                    .build();
+
+            StartLabelDetectionRequest labelDetectionRequest = StartLabelDetectionRequest.builder()
+                    .jobTag("DetectingLabels")
+                    .notificationChannel(channel)
+                    .video(vidOb)
+                    .minConfidence(50F)
+                    .build();
+
+            StartLabelDetectionResponse labelDetectionResponse = rekClient.startLabelDetection(labelDetectionRequest);
+            startJobId = labelDetectionResponse.jobId();
+
+            boolean ans = true;
+            String status = "";
+            int yy = 0;
+            while (ans) {
+
+                GetLabelDetectionRequest detectionRequest = GetLabelDetectionRequest.builder()
+                        .jobId(startJobId)
+                        .maxResults(10)
+                        .build();
+
+                GetLabelDetectionResponse result = rekClient.getLabelDetection(detectionRequest);
+                status = result.jobStatusAsString();
+
+                if (status.compareTo("SUCCEEDED") == 0)
+                    ans = false;
+                else
+                    System.out.println(yy +" status is: "+status);
+
+                Thread.sleep(1000);
+                yy++;
+            }
+
+            System.out.println(startJobId +" status is: "+status);
+        } catch(RekognitionException | InterruptedException e) {
+            e.getMessage();
+            System.exit(1);
+        }
+    }
+
+    public static void getLabelJob(RekognitionClient rekClient,
+                                   SqsClient sqs,
+                                   String queueUrl) {
+
+        List<Message> messages=null;
+        ReceiveMessageRequest messageRequest = ReceiveMessageRequest.builder()
+                .queueUrl(queueUrl)
+                .build();
+
+        try {
+            messages = sqs.receiveMessage(messageRequest).messages();
+
+            if (!messages.isEmpty()) {
+                //Loop through messages received.
+                for (Message message: messages) {
+                    String notification = message.body();
+
+                    // Get status and job id from notification.
+                    ObjectMapper mapper = new ObjectMapper();
+                    JsonNode jsonMessageTree = mapper.readTree(notification);
+                    JsonNode messageBodyText = jsonMessageTree.get("Message");
+                    ObjectMapper operationResultMapper = new ObjectMapper();
+                    JsonNode jsonResultTree = operationResultMapper.readTree(messageBodyText.textValue());
+                    JsonNode operationJobId = jsonResultTree.get("JobId");
+                    JsonNode operationStatus = jsonResultTree.get("Status");
+                    System.out.println("Job found in JSON is " + operationJobId);
+                    // Found job. Get the results and display.

```

Comment:
Add blank line before this line. I would say "Display information about job." as the print statement makes it obvious we have found a job.

Reply:
fixed